_docs_use-cases/use-case-consumption-get-attributes-of-peer.md

```diff
@@ -19,7 +19,7 @@ properties:
   - comments:
   - actor: Identity
   - component: Runtime
-  - trigger:
+  - trigger: REST API
   - precondition:
   - result:
   - priority: n/a
```

Comment:
api_route_regex: is missing -> possibly GET /api/v2/Attributes?{queryParameter}$